fem/fespace.hpp

```diff
@@ -378,7 +378,7 @@ class FiniteElementSpace
        The target quadrature points in the elements are described by the given
        QuadratureSpace, @a qs. */
    const QuadratureInterpolator *GetQuadratureInterpolator(
-      const QuadratureSpace &qs) const;
+      const QuadratureSpace &qs, const DofToQuad::Mode = DofToQuad::FULL) const;
```

Comment:
Same here, the new `DofToQuad::Mode` parameter is not necessary -- that is a settable property of the returned object.

Reply:
`done`